ncat: add examples (#12226)
tldr-pages/tldr: pages/linux/ncat.md

# ncat

> > Ncat is a general-purpose command-line tool for reading, writing, redirecting, and encrypting data across a network.

Reviewer: ```suggestion
> Read, write, redirect, and encrypt data across a network.
```

> > An alternative implementation of a similar utility called netcat/nc.

Reviewer 2: ```suggestion
> An alternative implementation of a similar utility called `netcat`/`nc`.
```

(or)

```suggestion
> An alternative implementation of a similar utility called Netcat/`nc`.
```

We enclose command names inside backticks. Alternatively for Netcat, we could capitalize the first letter since the project name follows the same.

> > Ncat is an alternative implementation of a similar utility called netcat/nc.

Reviewer: ```suggestion
> An alternative implementation of a similar utility called netcat/nc.
```

- Write output of specified file to the specified host on the specified port:

`ncat {{address}} {{port}} < {{path/to/file}}`

- Accept multiple incoming connections on an encrypted channel evading detection of traffic content:

`ncat --ssl -k -l {{port}}`

> - Connect to an open ncat connection over ssl:

Reviewer 2: ```suggestion
- Connect to an open Ncat connection over SSL:
```
(or)
```suggestion
- Connect to an open `ncat` connection over SSL:
```

We generally write protocols and abbreviations in upper-case.
